### ISSUE TITLE
FLO-2D - Special time-non-dependant quantities are not parsed

### ISSUE THREAD

**Issue by saberraz** [lutraconsulting/MDAL#411]
The max data are not parsed any more.

**Comment by FLO-2DKaren** [comment]
@saberraz 
Here is a test project.
https://flo-2d.sharefile.com/d-s381cae4e084e404fabe0e0f0a848b7e2

Mesh TS switch 1 6 min -  The mesh reads timdep.out and probably other ascii files.  
![image](https://user-images.githubusercontent.com/20424460/156607367-27e6dec6-a741-41c7-88bd-58cf46cdfbf4.png)

Mesh TS switch 2 6 min - The mesh reads hdf5.  No Maximums are available.
![image](https://user-images.githubusercontent.com/20424460/156607743-e371a3be-567d-44c5-a2c3-177fecf5f287.png)

I see that there are Max tables in the hdf5 output.  Can you use those in the Mesh Mapping tool?
![image](https://user-images.githubusercontent.com/20424460/156608006-74352e83-68cc-4f0f-b9f9-3239c614ff08.png)

**Comment by saberraz** [comment]
So, the issue is that Maximums are parsed correctly for FLO-2D ascii files but not the binary HDF format.

**Comment by PeterPetrik** [comment]
duplicate of https://github.com/lutraconsulting/MDAL/issues/382 ?

**Comment by vcloarec** [comment]
@FLO-2DKaren, the link with the test project doesn't work on y side.

**Comment by vcloarec** [comment]
I confirm duplicate #382.
The "Maxs" and "Mins" variable in the HDF5 file don't contain the aggregate values for each faces for all time steps, but the aggregate values for each time step for all faces, we have one value for each time step.

**Comment by FLO-2DKaren** [comment]
@vcloarec @saberraz, I agree that it's a duplicate.  I think we can solve this with the new mapping methods we discussed with Saber.  Why don't you close this comment and Comment 382.  We'll send Saber the full list of extra mapping stuff as soon as we get the new hdf5 results in the output.  Then Saber can get us a quote and we can start the development on your schedule.
